apps/src/accounts/AccountInformation/index.tsx

```diff
@@ -191,12 +192,21 @@ export const AccountInformation: React.FC<AccountInformationProps> = ({
   return (
     <>
       <hr />
-      <Heading2
-        visualAppearance="heading-sm"
-        className={commonStyles.sectionHeader}
-      >
-        {i18n.accountInformation_accountInformation()}
-      </Heading2>
+      <GlobalRegionWrapper
+        component={() => (
+          <Heading2
+            visualAppearance="heading-sm"
+            className={commonStyles.sectionHeader}
+          >
+            {i18n.accountInformation_accountInformation()}
+          </Heading2>
+        )}
+        componentId="AccountInfoHeader"
+        props={{
+          isInV1Navigaton: false,
+        }}
```

Comment:
This is all more or less reasonable!

Generally, I think it might be clearer if you actually made these components as `const` somewhere with the names you have given them and then it would just be `component={AccountInfoHeader}`, etc, for clarity.

If there are no props, just specify `props={{}}` for now. Unclear why vanilla JS does not like omitting the `props` value since it has a default... but that's not a big deal.